examples/metrics/src/main/java/io/opentelemetry/example/metrics/DoubleCounterExample.java

```diff
@@ -0,0 +1,57 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.metrics.DoubleCounter;
+import io.opentelemetry.metrics.LongCounter;
+import io.opentelemetry.metrics.Meter;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.Status;
+import io.opentelemetry.trace.Tracer;
+import java.io.File;
+import javax.swing.filechooser.FileSystemView;
+
+/**
+ * Example of using {@link DoubleCounter} and {@link DoubleCounter.BoundDoubleCounter} to count used space by file  with specific extension.
+ */
+public class DoubleCounterExample {
+
+  private static final Tracer tracer =
+      OpenTelemetry.getTracer("io.opentelemetry.example.metrics");
+  private static final Meter sampleMeter = OpenTelemetry.getMeterProvider()
+      .get("io.opentelemetry.example.metrics", "0.5");
+  private static final File directoryToCountIn = FileSystemView.getFileSystemView().getHomeDirectory();
+  private static final DoubleCounter methodCallCounter = sampleMeter
+      .doubleCounterBuilder("calculated_used_space")
+      .setDescription("should count space used by file extension")
```

Comment:
```suggestion
      .setDescription("Counts disk space used by file extension.")
```

It actually does so no need for the subjunctive 😉